### PR TITLE
[Draft][recipe] Add recipe for Gather/SparseToDense_AddV2/Cast pattern

### PR DESCRIPTION
**Pull request by BalyshevArtem**
This commit adds recipe for Gather/SparceToDense_AddV2/Cast pattern.

for https://github.com/Samsung/ONE/issues/8375#issuecomment-1070126309

ONE-DCO-1.0-Signed-off-by: Artem Balyshev a.balyshev@partner.samsung.com

### CODE REVIEW

File: res/TensorFlowLiteRecipes/Gather_SparseToDense_AddV2_Pattern_000/test.recipe [hunk 1]
```diff
@@ -0,0 +1,131 @@
+operand {
```
Comment (seanshpark) [inline res/TensorFlowLiteRecipes/Gather_SparseToDense_AddV2_Pattern_000/test.recipe:1]: Could you please rename starting with `Net_` ? And we can omit `_Pattern`

Comment (BalyshevArtem) [reply res/TensorFlowLiteRecipes/Gather_SparseToDense_AddV2_Pattern_000/test.recipe]: Done

**Comment by BalyshevArtem** [comment]
I will split this pr into two parts: the first - changes in `common-artifacts` and the second - changes in `recipe`

File: compiler/common-artifacts/exclude.lst [hunk 2]
```diff
@@ -73,6 +73,7 @@ tcgenerate(Neg_000)
 tcgenerate(Net_BroadcastTo_AddV2_001) # luci-interpreter doesn't support custom operator
 tcgenerate(Net_Conv_FakeQuant_000) # luci-interpreter doesn't support FakeQuant yet
 tcgenerate(Net_Dangle_001)
+tcgenerate(Net_Gather_SparseToDense_AddV2_000) # luci-interpreter doesn't support custom operator
```
Comment (BalyshevArtem) [inline compiler/common-artifacts/exclude.lst:76]: Add this due to luci-interpreter doesn't support custom AddV2

**Comment by BalyshevArtem** [comment]
Everything is merged, close it

**Pull request closed without merging**